tests/v2/codecoverage/rancherha/main.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import (
+	"github.com/rancher/rancher/tests/framework/clients/corral"
+	"github.com/sirupsen/logrus"
+)
+
+func main() {
+	results := corral.CorralConfigurations()
+	err := corral.SetupCorralConfig(results.CorralConfigVars)
+	if err != nil {
+		logrus.Fatalf("error setting up corral: %v", err)
+	}
+
+	configPackage := corral.CorralPackagesConfig()
+	path := configPackage.CorralPackagePath["aws-rke2-rancher-calico-v1.23.6-rke2r1-2.6.7"]
```

Comment:
Well this essentially a dumby value, I'm updating the rancher image after I have created it. Our custom rancher image, is always updated. The plan is to have a separate Jenkins job to build a new image every week. The scripts to do that already exist. In terms of k8s version, we can possible have this be config entry rather than it being hardcoded, as to keep up with k8s versions. I don't know if that a concern necessarily as are reporting code coverage not really validations if that makes sense.